paper-radio-group.html

```diff
@@ -120,6 +119,12 @@
       allowEmptySelection: {
         type: Boolean,
         value: false
+      },
+
+      disabled: {
+        type: Boolean,
+        reflectToAttribute: true,
```

Comment:
Just double checking like a nagging person: are you styling this based on `disabled` anywhere? (like `paper-radio-group[disabled[ {...}`. If not, this doesn't need to reflect